target/linux/lantiq/config-4.4

```diff
@@ -178,7 +179,6 @@ CONFIG_RTL8366_SMI=y
 CONFIG_SCHED_HRTICK=y
 # CONFIG_SCHED_INFO is not set
 # CONFIG_SCSI_DMA is not set
-# CONFIG_SENSORS_LTQ_CPUTEMP is not set
```

Comment:
@hauke I revert my statement.

CONFIG_SENSORS_LTQ_CPUTEMP depend on CONFIG_HWMON.

This is not enabled on target lantiq. Only in some of there subtargets for example xrx200. So CONFIG_SENSORS_LTQ_CPUTEMP could not be set with 
`make kernel_menuconfig CONFIG_TARGET=target ` only if i enable CONFIG_HWMON for all subtargets.

If you select xway in `make menuconfig` there are also some CONFIG parameters which will be removed. I would update this as well could you verify if this is ok? I have no subtarget xway.

subtarget falcon and xway_lecancy are ok no update is needed